templates/main.html

```diff
@@ -0,0 +1,84 @@
+<html>
```

Comment:
Please add todos for later commits:
1. Display rectangle with face detected.
2. Center the camera image.

Reply:
done